llvm/test/CodeGen/X86/atomic-float.ll

```diff
@@ -0,0 +1,15 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt < %s --mtriple=x86_64 --passes=atomic-expand -S -o - | FileCheck %s
```

Comment:
This belongs in test/Transforms/AtomicExpand/X86

Reply:
Are there missing codegen tests for this? Those should be added if they are missing